### PR TITLE
Module 10

### PR DESCRIPTION
**Pull request by DanielHare4**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: todo_app/data/mongodb_items.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        item = {
+            "id": str(len(list(self.collection.find())) + 1),
+            "name": title,
+            "status": "To Do"
+        }
+        self.collection.insert_one(item)
+
+    def change_item_status(self, id):
+        current_status = self.collection.find_one({"id": id})["status"]
+        new_status = "Done" if current_status == "To Do" else "To Do"
```
Comment (Jonesey13) [inline todo_app/data/mongodb_items.py]: (Not important but) might be worth extracting the item statuses into constants (or an Enum type)

File: .env.template
Comment (Jonesey13) [inline .env.template:1]: Just to mention that you still have Trello environment variables set on your Azure App Service.

**Review by Jonesey13 — APPROVED** [review]
Looks good to me! All appears to be working both locally and on Azure 👍

File: .env.template
Comment (DanielHare4) [reply .env.template:1]: Removed

**Pull request merged**